lrspline/__init__.py

```diff
@@ -80,6 +81,9 @@ def derivative(self, u, v, d=(1,1)):
         ])
         return retval.reshape(u.shape)
 
+    def __getitem__(self, idx):
```

Comment:
There's two different `__getitem__` here that do different things. I guess the controlpoint one is more useful, so maybe delete this one.

Reply:
There is only one `__getitem__` on LRSpline objects, right? The other one is for Basisfunctions. Returning controlpoints synergizes with what is in the `splipy`-library. Returning local knot vectors from Basisfunction objects might be more questionable, but it is what is done in the C-library and I guess the `BSplineBasis` from splipy as well, so it is consistent as well.

Reply:
Fair, I couldn't see the whole context in github. :-P